docs/deleteKeyValuePair.md

```diff
@@ -0,0 +1,131 @@
+<!--
+Licensed to the Apache Software Foundation (ASF) under one or more
+contributor license agreements.  See the NOTICE file distributed with
+this work for additional information regarding copyright ownership.
+The ASF licenses this file to You under the Apache License, Version 2.0
+(the "License"); you may not use this file except in compliance with
+the License.  You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+-->
+# Apache Accumulo Key-Value Pair Removal Example
+
+This is an example of how to remove a key-value pair
+
+    $ /path/to/accumulo shell -u username -p secret
+    username@instance> createtable deleteKeyValuePair
+    username@instance deleteKeyValuePair> insert 567890 name first Joe
+    username@instance deleteKeyValuePair> insert 567890 name last Smith
+    username@instance deleteKeyValuePair> insert 567890 address city Columbia
+    username@instance deleteKeyValuePair> scan
+    567890 address:city []    Columbia
+    567890 name:first []    Joe
+    567890 name:last []    Smith
+    username@instance deleteKeyValuePair> flush -w
+    2019-04-18 11:01:47,444 [shell.Shell] INFO : Flush of table deleteKeyValuePair completed.
+     
```

Comment:
Could use `tables` command in the shell to get the table id.  Could also show how to scan the metadata table in the shell to see the list of files Accumulo is currently using.

```
scan -t accumulo.metadata -c file -r 2<
```

Above command assumes the table id is `2`.

Reply:
Thank you!  This is what I was looking for.